### ISSUE TITLE
Feature: University Student model

### ISSUE THREAD

**Issue by lmz** [ActivitySim/activitysim#707]
Generate a synthetic population of university students and run them through a series of travel demand models specific to students.  ODOT has a couple university student models that make use of a university student travel survey.  

Added from `ActivitySim Budget Phase 9.xslx`

**Comment by lmz** [comment]
Comments from prior discussion:
- SFCTA: To confirm, is this about unviersity students living on-campus / in GQ?  Unviersity students living at home are already captured in the core models, and where appropriate the existing models are segmented to reflect unversity-specific choices (such as destination choice).  If this is about GQ university student behavior,  I think the default synthetic population for any Activitysim implementation should explicitly include university segment, and potenitally other non-institutional group quarters segments as well.  Input data on university GQ  is usually pretty easily available, althought survey data on GQ unviersity student travel behavior choices seems like it might be hard to find.

- SANDAG: we don't have a univ student model and have been considering adding one. The key challenge is we don't have a good estimate where univ students live (many live in GQs on campus, but many others don't).  For example, many SDSU students live in North Park and Pacific Beach whch are two hot hip neighbourhoolds that not close to SDSD campus.  Most of us have a sizable university populatioon in our regions?  If so, this could be a feature benenfit many of us.

- MTC:  Not a priority for us at this time.

**Comment by guyrousseau** [comment]
Thanks, Lisa for adding this. As I commented into the chat box of our meeting earlier this afternoon, regarding ID # 12, university student model, the standard approach appears to synthesize students at their place of residence, and then apply a school location model for assigning students to each campus. This type of model sometimes seems to struggle to assign students to the correct campus, which could result in incorrect model outcomes and/or over-specified model parameters that try to correct for the limitations of the school location model. Will this task synthesize students at their school location, and then assign them to a residential location? This way, students can be segmented by their type of residential location, which can include on-campus, off-campus student housing and rental properties, and regular family homes. The proportions of each type of residential arrangements would vary depending on the type of school, which can be controlled explicitly as well as used to find appropriate residential TAZs. Once this step is completed, this information would be used in the time of day, frequency, and mode choice models to improve their predictions, as well as the ActivitySim model's overall predictive analytics.

**Comment by JilanChen** [comment]
Would like to know the difference between what RSG has developed for SEMCOG (standard approach?) and what this new feature would include.

**Comment by jfdman** [comment]
A few things here:

1) First some background. University students are modeled as a specific person type in ActivitySim. University students living off-campus in family and non-family households are included in the synthetic population; most regions synthetic populations also include university students living in group quarters, which includes dormitories as well as fraternity and sorority houses.  Whether this is the case depends on whether the region builds separate group quarters populations or not. The university location choice model chooses a school zone for each student and each school tour generated by the student is sent to that zone.
2) A key problem with the current approach is that the ability of the population synthesis procedure to get university students living in the right location depends on the controls used. Typically group quarters populations are controlled by housing type: university\college, military, and other. The controls are available at the block level in decennial census data or from university inventories. Students living in non-GQ households are dependent upon whether age controls are used and at what geographic level, which varies between regions. As @guyrousseau mentions, one way to ensure that students are located accurately is to run a university student residential location choice model prior to running population synthesis.  This can then be used as a control to ensure that the right students are located in close proximity to major universities. An extension to this approach is to generate separate controls for each major university and tag the students with the university when they are generated, thus precluding the need to run school location choice for them. I believe this was used in the MAG model for ASU, we also did this for the LCOG model.
3) Another key problem with the current implementation is that ActivitySim only selects one zone for each student and sends all school tours to that zone. So if a campus spans many zones (e.g. University of Michigan spans dozens of zones across multiple campuses) then ActivitySim can't hope to capture inter/intra-campus travel correctly. This has implications for the ability of the model to generate the right number of walk, bike, and transit/campus shuttle trips. The solution we implemented for SEMCOG was to re-sample a school zone for each school activity based on classrooms and other space data by zone for U of M students. That way we could generate travel across campus and get trips onto university transit routes.
4) Another problem is that ActivitySim doesnt handle campus parking very well. The parking location choice model in ActivitySim is geared towards modeling downtown parking where the traveler tends to choose a parking spot close to work and walks. ActivitySim treats parking location as a special zone and travel between the lot and the activity location isn't explicitly modeled. Campus parking is quite different. The choice of parking can be limited based on type of pass owned, parking lot space constraints are more significant, and parking can be quite far from campus requiring a campus shuttle. For UMich we developed a simple parking allocation model for all auto trips onto campus where we insert an intermediate stop into the tour before and after the first and last on-campus activities respectively, and set the location of each stop equal to the parking zone, we then model all trips between the parking stops as if they are 'walk-transit' tours, so their choice of mode is limited to walk and walk-transit. In this way we get the mode right for parking that is right on campus and doesnt require a bus and we model transit trips for more distance parking. Its actually not a bad treatment for all parking stops including the use of peoplemover type modes and so-called 'park-and-hide' where travelers avoid parking charges by parking just outside the CBD, But this would require a larger-scale change to ActivitySim.
5) Household surveys tend to pick up students living in family households. They are not so good at recruiting students who live in non-family households, and most completely miss students living in group quarters. Students living in GQ are also not included in ACS data.  We do know they tend to own fewer cars, they tend to be the most active in terms of travel, they tend to have very high rates of walking, biking, and transit. But we need better data to accurately model them. Some agencies are  budgeting for targeted university student samples in their surveys and coordinating with university leadership to encourage participation via email lists and so on. Hopefully this will be more common in the future.

**Comment by i-am-sijia** [comment]
> As @guyrousseau mentions, one way to ensure that students are located accurately is to run a university student residential location choice model prior to running population synthesis. This can then be used as a control to ensure that the right students are located in close proximity to major universities. An extension to this approach is to generate separate controls for each major university and tag the students with the university when they are generated, thus precluding the need to run school location choice for them. I believe this was used in the MAG model for ASU, we also did this for the LCOG model.

Yes. Confirm this is how the MAG major university model was designed and implemented. After MAG, we also implemented the major university model for Ohio DOT's 3C (Columbus, Cincinnati, Cleveland) model. The MAG and Ohio major university model also simulates if the student is employed on campus and if yes, sets the campus zone as the work location.
